autoload/asyncomplete/sources/flow.vim

```diff
@@ -44,7 +44,7 @@ function! s:handler(opt, ctx, params, id, data, event) abort
         if a:data == 0
             let l:res = json_decode(a:params['stdout_buffer'])
             if !empty(l:res) && !empty(l:res['result'])
-                let l:matches = map(l:res['result'], '{"word": v:val["name"], "dup": 1, "icase": 1, "menu": "[Flow]"}')
+                let l:matches = map(l:res['result'], '{"word": v:val["name"], "dup": 1, "icase": 1, "menu": "[Flow]    " . v:val["type"]}')
```

Comment:
we need to put this under a config value similar to `prefer_local`. Some people might not prefer this.

Reply:
Sounds good, ill work on that addition tonight.